full_snapshot.proto

```diff
@@ -383,6 +383,15 @@ message RelationInformation {
     NullString default_value = 4;
     bool not_null = 5;
     int32 position = 6;
+    repeated ColumnStatistic statistics = 7;
```

Comment:
Hmm, is this repeated due to `inherited`?

Reply:
Yes, the parent table will have stats for both `inherited = true` (includes child tables) and `inherited = false` (excludes child tables). We decided to store both for now.

Reply:
Ok, I think that works--we could also do non-repeated `statistics` and `inherited_statistics` if it's just the two.